index.html

```diff
@@ -0,0 +1,275 @@
+                                    <p>JavaScript</p>
+                                </div>
+                            </li>
+                        </ul>
+                    </div>
+                <div class="normal-button">
+                    <p>See project</p>
+                </div>
+            </div>
+            <div class="card">
+                <div id="card-img-3"></div>
+                <h2 id="card-title">Multi-Post Stories</h2>
+                <div id="card-info" class="row-container">
+                    <h3>CANOPY</h3>
+                    <i class="fa-solid fa-circle"></i>
+                    <p>Back End Dev</p>
+                    <i class="fa-solid fa-circle"></i>
+                    <p>2015</p>
+                </div>
+                <p id="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                    <div id="card-tech">
+                        <ul id="list-categories" class="row-container">
+                            <li>
+                                <div class="box-background">
+                                    <p>HTML</p>
+                                </div>
+                            </li>
+                            <li>
+                                <div class="box-background">
+                                    <p>CSS</p>
+                                </div>
+                            </li>
+                            <li>
+                                <div class="box-background">
+                                    <p>JavaScript</p>
+                                </div>
+                            </li>
+                        </ul>
+                    </div>
+                <div class="normal-button">
+                    <p>See project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="about-me">
+        <div class="column">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                contact me.</p>
+            <div id="social-media-icons">
+                <p>LET'S CONNECT</p>
+                <ul>
+                    <li> <a href="#"><i class="fa-brands fa-github"></i></a></li>
+                    <li> <a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li> <a href="#"><i class="fa-regular fa-hand-peace fa-flip-horizontal"></i></a></li>
+                    <li> <a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li> <a href="#"><i class="fa-sharp fa-solid fa-m"></i></a></li>
+                </ul>
+            </div>
+            <div class="normal-button">
+                <p>Get my resume</p>
+            </div>
```

Comment:
Well done on the project 💯 but the button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design) 👇🏽 

![Screenshot 2023-04-27 at 23 29 45](https://user-images.githubusercontent.com/45739963/235047705-d9059548-e8b6-4d78-bab7-b367fee1b0ef.png)